lib/platform/bitbucket-server/index.md

```diff
@@ -8,6 +8,9 @@
 ## Features awaiting implementation
 
 - Creating issues not implemented yet, used to alert users when there is a config error
+- The `automergeStrategy` configuration option has not been implemented for this platform, and all values behave as if
+  the value `auto` was used. Renovate will implicitly use the merge strategy configured as 'default' in the Bitbucket
+  Server repository itself, and this cannot be overridden yet.
```

Comment:
```suggestion
- The `automergeStrategy` configuration option has not been implemented for this platform, and all values behave as if the value `auto` was used. Renovate will implicitly use the merge strategy configured as 'default' in the Bitbucket Server repository itself, and this cannot be overridden yet
```

We use one sentence per line pattern in our `*.md` files. 😉
We also drop the final `.` on a ordered/unordered list.